etc/kayobe/kolla/kolla-build.conf

```diff
@@ -4,9 +4,12 @@
 {# This tag should be updated when Ubuntu package repo snapshot versions are changed. #}
 {% if kolla_base_distro == 'ubuntu' %}
 base_tag = noble-20241118.1
+apt_preferences = /home/runner/_work/stackhpc-kayobe-config/stackhpc-kayobe-config/src/kayobe-config/etc/kayobe/environments/ci-builder/builder-apt-preferences
```

Comment:
Is it possible to use variables to target this file? Such as ``KAYOBE_CONFIG_PATH`` (env var) or ``kayobe_config_env_path`` (kayobe var)

Reply:
Yep, looks like `KAYOBE_CONFIG_PATH` works, good idea!

https://github.com/stackhpc/stackhpc-kayobe-config/actions/runs/15681228073/job/44172920042